website_and_docs/content/blog/2022/end-of-year-review.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Year End Review 2022"
+linkTitle: "Year End Review 2022"
+date: 2022-12-20
+tags: ["selenium"]
+categories: ["general"]
+author: Corina Pip ([@imalittletester](https://twitter.com/imalittletester))
+description: >
+  Looking back at our achievements from 2022
+---
+
+It was an amazing year here at the Selenium project, and we wanted to remind you of some of the great things we accomplished.
+
+In terms of releases, we continued to build on top of Selenium 4 which was launched in October 2021. This year we
+published [4.2.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.2.0),
+[4.3.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.3.0), [4.4.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.4.0),
+[4.5.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.5.0), [4.6.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.6.0)
+and [4.7.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.7.0).
+These releases included improvements, bug fixes, removal of deprecated functionality, improved error handling,
+updates of the underlying libraries, support for the latest DevTools/BiDi
```

Comment:
"BiDi" the feature is not dependent on Chrome versions, so for this section it should be just "Chrome DevTools versions"